src/pages/products/products.js

```diff
@@ -10,6 +10,15 @@ import { useAuth } from '../../filter-context/auth-context'
 import { Navigate, useNavigate } from 'react-router-dom'
 import axios from 'axios'
 import { useWishlist } from '../../filter-context/wishlist-context'
+import { addToCartHandler } from '../../util-functions/add-to-cart'
+import {grocery} from '../../filter-functions/grocery'
+import {fashion} from '../../filter-functions/fashion'
+import { electronics } from '../../filter-functions/electronics'
+import { home } from '../../filter-functions/home'
+import { appliances } from '../../filter-functions/appliances'
+import { mobile } from '../../filter-functions/mobile'
+import { clearAll } from '../../filter-functions/clearAll'
+import { useCategory } from '../../filter-context/category-context'
```

Comment:
put it into single index file then import export

Reply:
ok